projects/view/client/rtProject/rtProjectInfoSection.css

```diff
@@ -0,0 +1,215 @@
+.rtProjectInfoSection .section { 
+	max-width: 1024px;
+	clear: both;
+	padding: 0px;
```

Comment:
Remove `px` as it is not needed for zero values. Also, use relative units such as `em` or `rem` for layout related style rules.